RELEASE_NOTES.md

```diff
@@ -11,6 +11,7 @@ A new header is inserted each time a *tag* is created.
 
 ## v1.23.3
 
+- ImGuiHelper: add support for Y flip.
```

Comment:
Are you cherry-picking this into v1.23.3? If not, this should go under `main branch`.

Reply:
Yes I intend to do a cherry-pick to v1.23.3.

Reply:
Sounds good!